docs/apis/programming_guide.md

```diff
@@ -440,6 +440,46 @@ The `execute()` method is returning the `JobExecutionResult`, including executio
 accumulator results. `print()` and `collect()` are not returning the result, but it can be
 accessed from the `getLastJobExecutionResult()` method.
 
+Further, `collect`, `print`, `printToErr` and `count` methods are *eager execution* methods. This
```

Comment:
I think parenthesis make these easier to recognize as methods when skimming over text.